R/annotate_ssm_motif_context.R

```diff
@@ -0,0 +1,191 @@
+#' @title Annotate mutations target AID motif
```

Comment:
Also, this is **all** migrated code (i.e. no new code added to this function, right?)

Reply:
Usually, I have different commits to migrate a function to a package and to make changes to it. In this case, it seems that I included the function and changed its examples in [this](https://github.com/morinlab/GAMBLR.results/pull/10/commits/2a05de242d3aec163b53445e4a947ee282b45cff) same commit (sorry, I see this is not a good practice). After that, I changed IRanges and GenomicRanges imports separately in [this](https://github.com/morinlab/GAMBLR.results/pull/10/commits/5a23ebf618fc36e12debca02779640637bc689ea) commit. 

Is there a better way to migrate a function to a package while letting reviewers know if the code was changed or not?

Reply:
Thanks for clarifying, see the conversation in Slack gamblr-help for more info on how to deal with migrating functions in the future :)